### ISSUE TITLE
[Feature request] Pass jobId as variable

### ISSUE THREAD

**Issue by andrewetchen** [peckjon/github-to-guru#6]
Hi Jon,

First and foremost, thank you so much for taking the time to build this action, I wish I had found this sooner!
I have unintentionally become quite knowledgeable with the GetGuru web app having used it quite extensively the last 2 years.

### Context

Our current setup is to send your actions' workflow status to our teams’ Slack channel using a Slack Webhook. However, sending the status of your action will often result in a “success” status even if syntax errors exist within `.yaml` files for example.

### Feature Request

Pass the `“jobId”` as an `env` variable which can then be used in a sequential job after your action:

Reference to `cURL` within the [index.js](https://github.com/peckjon/github-to-guru/blob/master/index.js):

https://github.com/peckjon/github-to-guru/blob/98392ff55ace69033837de843cbba0737e6dc8ad/index.js#L22-L23

![image](https://user-images.githubusercontent.com/28404165/112903047-03abc080-90b5-11eb-9459-cfd96e2ae5c0.png)

Being able to message our Slack channel with the status results from the following `POST` would be ideal:

```yaml
run: |
  curl -u $GURU_USER_EMAIL:$GURU_USER_TOKEN https://api.getguru.com/api/v1/import/{{ jobId }}/status
```

... if `"migrationError'` is present/detected, send "fail" status and include the error message value:

![image](https://user-images.githubusercontent.com/28404165/112902877-c9daba00-90b4-11eb-8b60-3ec69da83945.png)

Thanks in advance and I appreciate your time.

**Comment by peckjon** [comment]
Thanks for the feature request @andrewetchen !

Resolved in release [1.6.0](https://github.com/peckjon/github-to-guru/releases/tag/1.6.0)

Note that I've provided the full API response in $GURU_SYNC_RESPONSE in case the definition gets extended, so just post-process that var's contents as JSON

**Comment by andrewetchen** [comment]
Thank you so much for actioning that so quickly! Much appreciated, @peckjon
